model_saver.py

```diff
@@ -0,0 +1,28 @@
+import joblib
+import logging
+
+
+class MovieSaver:
+    def __init__(self):
+        self.logger = self.setup_logger()
+
+    @staticmethod
+    def setup_logger():
+        logger = logging.getLogger('MovieSaver')
+        logger.setLevel(logging.INFO)
+        handler = logging.StreamHandler()
+        formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+        handler.setFormatter(formatter)
+        logger.addHandler(handler)
+        return logger
+
+    @staticmethod
+    def save_model(model, model_path):
```

Comment:
why make it static again? 
you're already creating an instance to call the method
```
saver = MovieSaver()
saver.load_model(model_path)
```
it doesn't need to be static in this case